health/micro-ui/web/micro-ui-internals/packages/css/src/pages/employee/healthdss.scss

```
@@ -1407,4 +1412,210 @@
+    gap: 8px;
+    align-items: center;
+
+    .digit-tag-wrapper{
+      height: fit-content;
+    }
+  }
+
+  &__header-with-tag-header {
+    font-family: "Roboto Condensed";
+    font-weight: 700;
+    font-style: Bold;
```

Stylelint (17.6.0)

[error] Expected "Bold" to be "bold" (value-keyword-case)

(value-keyword-case)